### ISSUE TITLE
Event in service on configuration changed

### ISSUE THREAD

**Issue by ipvadimmironov** [Red-Folder/bgs-core#87]
Hello,
If there any event fired up in service on configuration changed ( when `setConfiguration` has executed in frontEnd) ?

Or if there any other way to communicate between front and back?

**Comment by Red-Folder** [comment]
Yes the setConfiguration (frontend) calls through to the setConfig in the service

See https://github.com/Red-Folder/bgs-core/wiki/Using-the-configuration for a summary.
